README.md

```diff
@@ -73,11 +73,11 @@ Generally, it is recommended to use the latest release of kube-state-metrics. If
 
 | kube-state-metrics | Kubernetes client-go Version |
 |--------------------|:----------------------------:|
-| **v2.4.2**         | v1.23                        |
 | **v2.5.0**         | v1.24                        |
 | **v2.6.0**         | v1.24                        |
 | **v2.7.0**         | v1.25                        |
 | **v2.8.1**         | v1.26                        |
+| **v2.9.0**         | v1.26                        |
```

Comment:
Further below there's `registry.k8s.io/kube-state-metrics/kube-state-metrics:v2.8.1` which should be updated as well (I forgot with the v2.8.2 release)